google-cloud-clients/google-cloud-spanner/src/main/java/com/google/cloud/spanner/spi/v1/GapicSpannerRpc.java

```diff
@@ -348,7 +345,20 @@ public void deleteInstance(String instanceName) throws SpannerException {
             .setOperationId(MoreObjects.firstNonNull(updateId, ""))
             .build();
     GrpcCallContext context = newCallContext(null, databaseName);
-    return databaseStub.updateDatabaseDdlOperationCallable().futureCall(request, context);
+    OperationCallable<UpdateDatabaseDdlRequest, Empty, UpdateDatabaseDdlMetadata> callable = databaseAdminStub.updateDatabaseDdlOperationCallable();
+    OperationFuture<Empty, UpdateDatabaseDdlMetadata> operationFuture = callable.futureCall(request, context);
+    try {
+      operationFuture.getInitialFuture().get();
+    } catch (InterruptedException e) {
+      // The error would be handled in SpannerImpl, swallow it here
```

Comment:
It's not a good practice to swallow `InterruptedException`:

https://stackoverflow.com/questions/18682333/why-should-we-not-swallow-the-interruptedexception

Reply:
I'll be OK with just re-throwing this.